README.rst

```diff
@@ -16,60 +16,99 @@ between a set of devices owned by a same account.
 API
 ===
 
-* `API Design proposal
-  <https://github.com/mozilla-services/readinglist/wiki/API-Design-proposal>`_
 * `Online documentation <http://readinglist.readthedocs.org/en/latest/>`_
 
 
 Run locally
 ===========
 
-By default, readinglist persists its records inside a `Redis
-<http://redis.io/>`_  database, so it has to be installed first (see the
-"Install Redis" section below for more on this).
+*Reading List* is based on top of the `cliquet <https://cliquet.rtfd.org>`_ project, and
+as such, please refer to cliquet's documentation for more details.
 
-Once Redis is installed:
+
+For development
+---------------
+
+By default, *Reading List* persists the records and internal cache in a PostgreSQL
+database.
+
+The default configuration will connect to the `postgres` database on
+`localhost:5432`, with user/password `postgres/postgres`. See more details
```

Comment:
You are right - rsT use the ** or * or ``.  simple tick is used only in cunjunction with some extra markers for links for example.

http://docutils.sourceforge.net/docs/user/rst/quickstart.html#text-styles